statsmodels/compat/python.py

```diff
@@ -20,7 +20,7 @@
     pickle = cPickle
     import urllib.request
     import urllib.parse
-    from urllib.request import HTTPError, URLError
+    from urllib.error import HTTPError, URLError
```

Comment:
In py36 and py37 these are the same objects

Reply:
I suspected, but this seems like a better import location.